src/rest.rs

```diff
@@ -1122,14 +1148,116 @@ fn handle_request(
                 .map_err(|err| HttpError::from(err.description().to_string()))?;
             http_message(StatusCode::OK, txid.to_hex(), 0)
         }
+        (&Method::GET, Some(&"txs"), Some(&"outspends"), None, None, None) => {
```

Comment:
This one is missing the internal prefix.

Reply:
intentionally, as this one will be called directly from the front end (but limited to 50 txids per request, and replacing a more expensive solution)